RemoteCarControllerClient/app/src/main/java/elsys/org/remotecarcontrollerclient/MainActivity.java

```diff
@@ -0,0 +1,37 @@
+package elsys.org.remotecarcontrollerclient;
```

Comment:
Shouldn't be the package named org.elsys.remote_control_car.client, to be relatively the same as the backend packages?